.gitignore

```diff
@@ -5,6 +5,7 @@
 # gitignore template for the CypressIO, browser test framework
 # website: https://www.cypress.io/
 
+cypress/
```

Comment:
the modification should be more specific, you should add two paths:
- `cypress/e2e/1-getting-started/*`
- `cypress/e2e/2-advanced-examples/*`

Reply:
I added those folders to .gitignore file
Thx for you advise